cmake/bintray_deploy.json.in

```diff
@@ -0,0 +1,28 @@
+{
+    "package": {
+        "name": "Cockatrice-dev",
```

Comment:
This should be -git I think, Dev packages are meant to only include headers. If it's not git then -devel